jtl/functions.go

```diff
@@ -639,6 +642,48 @@ func fnCurl(ctx Context, doc *JDoc, params []string) interface{} {
 	}
 }
 
+// fnCurlOAuth1 provides curl-like functionality to reach out to helper web services, with oauth 1.0 authenication
+func fnCurlOAuth1(ctx Context, doc *JDoc, params []string) interface{} {
+	stats := ctx.Value(EelTotalStats).(*ServiceStats)
+	if params == nil || len(params) < 2 || len(params) > 6 {
```

Comment:
SHouldn't the check be len(params) == 6?  Ie, I think we should throw an error if the parameter is not exactly equal to 6?